src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -41,6 +41,7 @@ public override void SetVirtualView(IView view)
 		// TODO: NET8 issoto - Change the return type to MauiAppCompatEditText
 		protected override void ConnectHandler(AppCompatEditText platformView)
 		{
+			platformView.ViewAttachedToWindow += OnViewAttachedToWindow;
```

Comment:
Curious to hear why set the return type on this event instead of the constructor?

Reply:
I think is just normal practice also if it gets reused you know the value is updated when being added